Files sshguard-0.91/.sshguard.c.swp and sshguard-0.91-pam/.sshguard.c.swp differ
diff -urN sshguard-0.91/sshguard.h sshguard-0.91-pam/sshguard.h
--- sshguard-0.91/sshguard.h	2007-02-10 17:00:47.000000000 +0200
+++ sshguard-0.91-pam/sshguard.h	2007-03-12 07:57:33.000000000 +0200
@@ -17,8 +17,8 @@
  *
  */
 /* regex for log entries representing brute force trials */
-#define SSHCRACK_REGEX      "sshd\\[[0-9]+\\]: (Failed|Illegal user|Invalid user|User) .+from ((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]([0-9])?)(\\.(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]([0-9])?|0)){3})( not allowed)?"
+#define SSHCRACK_REGEX	"sshd\\[[0-9]+\\]: (Failed|Illegal user|Invalid user|User|pam_unix\\(sshd:auth\\): authentication failure;).+(from |rhost=)((25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]([0-9])?)(\\.(25[0-5]|2[0-4][0-9]|1[0-9]{2}|[1-9]([0-9])?|0)){3})( not allowed)?"
 /* field number in SSHCRACK_REGEX containing the attacker IP address */
-#define REGEXIPENTRY        2
+#define REGEXIPENTRY        3
 
 #endif
